Rambler SSP ad network

CrazySquirrel opened the pull request:
Hi.
I added support for a new ad network "Rambler SSP".
There are new type for amp-ad tag, example of usage and description for it.

jridgewell commented:
/to @lannka

lannka commented on ads/capirs.md:
````diff
@@ -0,0 +1,39 @@
+    data-begun-block-id="5665768"
+    json=""
+>
+```
````
please complete the example with a closing tag.

lannka commented on examples/capirs.amp.html:
```diff
@@ -0,0 +1,91 @@
+<body>
+<h1>Rambler SSP ad</h1>
+
+<amp-ad layout="responsive" width="100" height="100" type="capirs" data-begun-auto-pad="434906118" data-begun-block-id="434908944" data-override-width="1000" data-override-height="1000"
```
please move your example into `/examples/ads.amp.html`

lannka commented:
@CrazySquirrel please ping the thread when you're ready for the next round of review.

CrazySquirrel commented:
@lannka OK, I'm definitely ready. I implemented the rendering API as you asked. About the nitpick, I don't quite understand what you meant. I guess that I have a different settings for code formatting in my IDE. But that's OK, I've already changed it, although It would be great to have some file with code formatting settings for IDE in repository.

lannka commented on ads/capirs.js:
```diff
@@ -0,0 +1,57 @@
+        const reportId = 'capirs-' + banner['banner_id'];
+        window.context.reportRenderedEntityIdentifier(reportId);
+      },
+      unexist: () => {
```
would the following work?
```
unexist: window.context.noContentAvailable
```

CrazySquirrel replied:
Yes. It probably will work.

lannka commented on examples/ads.amp.html:
```diff
@@ -1024,14 +1025,24 @@ <h2 id="purch">Purch 300x250</h2>
   </amp-ad>
 
+  <h2 id="capirs">Rambler&Co</h2>
+  <amp-ad width="500" height="250"
```
the slot loads an ad of height 90. do you want to tune the dimension to make it looks nicer?

CrazySquirrel replied:
The thing is that the size of the advertisement is not known until it is loaded.
In real placements it is expected to customize the advertising and the place in such a way that they sizes will be equal or dynamic.
In the example, I can specify the size as auto. This will work, but I don't know will it be better or not.

lannka commented on examples/ads.amp.html:
```diff
@@ -685,17 +686,17 @@ <h2 id="flite">Flite</h2>
-  <h2 id="fusion">Fusion</h2> 
-  <amp-ad width=600 height=100 
+  <h2 id="fusion">Fusion</h2>
+  <amp-ad width=600 height=100
```
since you're touching the line, could you help to add double quote around the values?

CrazySquirrel replied:
Done
